README.md

```diff
@@ -38,9 +38,9 @@ docker compose up
 
 ### Windows
 
-In order to user Docker on Windows you must install [Docker desktop for Windows](https://docs.docker.com/desktop/windows/install/)
```

Comment:
it's not required to install docker for windows ?

Reply:
Actually 'Docker desktop' is an option/suggestion not a requirement, there is other way to use docker on windows on top of WSL2.   Furthemore the subscription terms for Docker desktop has changed and I'm not sure that *requiring* is appropriate.